tests/test_fis.py

```diff
@@ -44,3 +44,47 @@ def test_fis_tag(self):
             'experimentTemplate'
         )
         assert experiment['tags'] == {'Location': 'Messina'}
+
+    def test_fis_mark_match(self):
+        session_factory = self.replay_flight_data(
+            "test_fis_mark_match"
+        )
+        p = self.load_policy(
+            {
+                "name": "test_fis_mark_for_op",
+                "resource": "aws.fis-template",
+                "filters": [{'tag:Name': 'Shiney'}],
+                "actions": [
+                    {
+                        "type": "mark-for-op",
+                        "days": 4,
+                        "op": "delete",
+                        "tag": "custodian_cleanup",
```

Comment:
it looks like the first recording (list/get experiment) already has this value, so its not evident that the action is working, it would be good to put an explicit check here.

Reply:
Fixed api calls recording. Thanks !